modin/pandas/base.py

```diff
@@ -2820,6 +2820,7 @@ def sort_values(
                 na_position=na_position,
             ).index
             if inplace:
+                self.reset_index(drop=True, inplace=True)
```

Comment:
This will be fixed by #1754.

Reply:
Ok I added this test to branch in #1754 . However it may take some time to merge it while here fix is simple as one line of code. We could merge it while your branch is under review.